shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -153,16 +153,24 @@ @implementation FlutterDartProject {
 #pragma mark - Override base class designated initializers
 
 - (instancetype)init {
-  return [self initWithPrecompiledDartBundle:nil];
+  return [self initWithPrecompiledDartBundle:nil dartVmArgs:nil];
 }
 
-#pragma mark - Designated initializers
+- (instancetype)initWithDartVmArgs:(nullable NSArray<NSString*>*)args {
+  return [self initWithPrecompiledDartBundle:nil dartVmArgs:args];
+}
 
 - (instancetype)initWithPrecompiledDartBundle:(nullable NSBundle*)bundle {
+  return [self initWithPrecompiledDartBundle:bundle dartVmArgs:nil];
+}
+
+#pragma mark - Designated initializers
+
+- (instancetype)initWithPrecompiledDartBundle:(nullable NSBundle*)bundle dartVmArgs:(nullable NSArray<NSString*>*)args {
   self = [super init];
 
   if (self) {
-    _settings = DefaultSettingsForProcess(bundle);
+    _settings = DefaultSettingsForProcess(bundle, args);
   }
 
   return self;
```

Comment:
gpus_ReturnNotPermittedKillClient crash on iOS [62403](https://github.com/flutter/flutter/issues/62403) Can you help us have a look，please？